tests/filecheck/dialects/transform/transform_ops.mlir

```diff
@@ -25,8 +25,8 @@
 }) : () -> ()
 
 %input = "test.op"() : () -> !transform.any_value
-// CHECK: %tiled_op, %loop_op, %remainder = "transform.structured.tile_using_for"(%input) <{static_sizes = array<i32: 8, 8>}> : (!transform.any_value) -> (!transform.any_op, !transform.any_op, !transform.any_op)
```

Comment:
https://mlir.llvm.org/docs/Dialects/Transform/#attributes-73
Upstream docs suggest this should be `i64`